src/styles/index.scss

```diff
@@ -2,2 +1,236 @@
+$cardFont: Roboto, Arial, sans-serif;
+$defaultFontWeight: 400;
+$accentColor: #060b35;
+$secondaryColor: #616070;
+$moyoBlue: #00acdc;
+$transitionTime: 300ms;
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-align: left;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* #region Header Styles */
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+.header__logo {
+  display: block;
+}
+
+.nav {
+  display: flex;
+  list-style: none;
+}
+
+.nav li {
+  display: block;
+  margin-left: 20px;
+  position: relative;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: $accentColor;
+  padding: 23px 0;
+  text-transform: uppercase;
+  transition: color $transitionTime;
+}
+
+.nav__link:hover {
+  color: $moyoBlue;
+}
+
+.nav__link--is--active {
+  color: $moyoBlue;
+}
+
+.nav__link--is--active::after {
+  content: '';
+  position: absolute;
+  bottom: -23px;
+  display: block;
+  background-color: $moyoBlue;
+  height: 4px;
+  width: 100%;
+  border-radius: 2px;
+}
+
+/* #endregion */
+
+/* Region grid */
+
+// .catalog {
```

Comment:
I see that you have commented out some code here. It's a good practice to remove commented out code to keep the codebase clean. If you need this code in the future, you can always get it back from the version control system.